shelley/design-spec/delegation_design_spec.tex

```diff
@@ -2644,31 +2659,55 @@ \subsection{Reminder: Stakepool Registration}
 \subsection{Epoch Rewards}
 \label{epoch-rewards}
 
-There will be four sources of rewards for an epoch: \emph{transaction
-  fees}, non-refundable parts of \emph{deposits}, \emph{monetary
-  expansion}, and rewards from the \emph{previous} epoch.
+There will be three sources of rewards for an epoch: \emph{transaction
+  fees}, non-refundable parts of \emph{deposits}, and \emph{monetary
+  expansion}.
 
 \subsubsection{Transaction Fees}
 \label{transaction-fees-1}
 
 All transaction fees from all transactions from all blocks created
-during the epoch will be used as rewards.
+during the epoch will be added to the rewards pot of that epoch.
 
 \subsubsection{Deposits}
 \label{deposits-1}
 
 As explained in \cref{fees-deposits}, a part of the deposits for
 certificates and UTxO entries are non-refundable, and contribute to
-the rewards pool instead.
+the rewards pot instead.
 
 \subsubsection{Monetary Expansion}
 \label{monetary-expansion}
 
-Let \(T\) be the total amount of ADA in existence during a specific
-epoch, and let \(T_\infty\) be the maximal possible amount of ADA in the
-future. At this moment, \(T=31,000,000,000\) and
-\(T_\infty=45,000,000,000\). Then the amount of
-\(\rho\cdot(T_\infty - T)\) ADA will be newly created.
+Every epoch, the total amount of ada in circulation \(T\) will be increased by
+adding ada to the rewards pot. To ensure the amount of ada never exceeds a
+finite, specified limit \(T_\infty\), the increase will reduce exponentially.
```

Comment:
It doesn't belong to this PR probably, but I find it a bit confusing that we use the infinity symbol for a finite value :thinking:

Reply:
It's the amount of ada in the limit time -> infinity.